sys-apps/flashrom/flashrom-9999.ebuild

```diff
@@ -1,15 +1,17 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit toolchain-funcs
 if [[ ${PV} == "9999" ]] ; then
 	EGIT_REPO_URI="https://review.coreboot.org/flashrom.git"
 	inherit git-r3
 else
-	SRC_URI="https://download.flashrom.org/releases/${P}.tar.bz2"
+	MY_P="${PN}-v${PV}"
+	SRC_URI="https://download.flashrom.org/releases/${MY_P}.tar.bz2"
 	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~sparc ~x86"
```

Comment:
I know it's not done by you, but live ebuilds shouldn't have KEYWORDS set.

Reply:
`=flashrom-9999` looks be effectively masked despite of `KEYWORDS`. I think there was a good intention in adding `KEYWORDS`: easy bumping by copying `9999` ebuild.

Could leaving `KEYWORDS` as it is be tolerated?

Reply:
I'm a bit at awe that nothing seems to be yelling about it (pkgcheck or repoman) and devmanual isn't strict about it, but PMS is: 
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-650007.3.3

I don't really mind either way, but let's respect the PMS on this one, and devmanual guidelines. Ie, the code is changing and cant be tested, so it can't be trusted on all these arches.

Reply:
I think it's because this document is not contain any explanation of a purpose of `9999` ebuilds.
`9999` ebuilds are extremely experimental so their `unsupportness` overwhelms any other indicators.

PMS is a subject to amendment as I see.

Reply:
I can comment line if necessary though as a compromise.

Reply:
I was gonna suggest that, too ;)

Reply:
Just a moment. `KEYWORDS` assignment dwells in the else block. `KEYWORDS` are not set if `${PV} == "9999"`.

This original wit design is made by initial author. It looks cute for me.

Reply:
Hmm, right.